.github/workflows/build.yml

```diff
@@ -19,9 +19,13 @@ jobs:
         uses: actions/checkout@v2
       - name: Use Xcode 12.4
         run: sudo xcode-select -switch /Applications/Xcode_12.4.app
+      - name: Install xcpretty
```

Comment:
Random - when I was trialing testflight stuff via GH Actions, I didn't need to install `xcpretty` to use it. 🤷‍♀️

Reply:
Oooh, good to know! I'll try removing it.